config.toml

```diff
@@ -37,17 +37,17 @@ pygmentsStyle = "friendly"
     languageCode = "zh-CN"
     weight = 2
     [[languages.zh.menu.global]]
-      name = "新闻"
+      name = "??????"
```

Comment:
Here is lost something. @jstrachan please help to locate the issues.

Reply:
ah - looks like a bug in updatebot not correctly handling the character encoding, I'll take a look

Reply:
@LinuxSuRen restored the Chinese text now - this is the bug https://github.com/jenkins-x/updatebot/issues/19

have a fix - just releasing it...